notebooks/egov/pgr-create-curl.txt

```diff
@@ -0,0 +1,17 @@
+curl 'https://staging.digit.org/pgr-services/v2/request/_create?tenantId=pg.citya' \
+  -H 'authority: staging.digit.org' \
+  -H 'accept: application/json, text/plain, */*' \
+  -H 'accept-language: en-GB,en-US;q=0.9,en;q=0.8' \
```

Comment:
Please put only notebooks in this directory

Reply:
done